tutorials/modular-load-balancing-with-terraform/index.md

```diff
@@ -119,6 +119,8 @@ In this tutorial, you run all commands by using the [Google Cloud Shell](https:/
 
         cd terraform-google-examples
 
+        git submodule init && git submodule update
```

Comment:
You can add `--recursive` to the `git clone` command above instead.

Reply:
@ndmckinley , is that a better way of accomplishing Betsy's intended result or just an alternative way?

Reply:
It's a bit more standard, and I find it aesthetically better, but it's nearly identical.  There are tiny differences, none of which apply today.  If, for instance, we ever added any submodules to Terraform, `--recursive` would handle that case, but this would ... probably not?  I think?  By my read of the manpages?  I certainly don't insist on this change.

Reply:
Thanks. I'll leave it to Betsy to decide what to do with this feedback.